ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/stun.schema.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  stun:
+    type: dict
+    keys:
+      client:
+        type: dict
+        description: |
+          "If a device is a client"
```

Comment:
@burnyd Please remove the extra quotes in the descriptions. Looks like we are referencing a quote in the rendered documentation